bats/tests/helpers/vm.bash

```diff
@@ -142,14 +142,6 @@ start_application() {
     if using_docker; then
         wait_for_container_engine
     fi
```

Comment:
Line 102: still has `"version": 7,`

Reply:
```
branch: bats-improvements {7} → $ cat -n bats/tests/helpers/vm.bash | sed -n 95,105p
    95	    wsl_integrations="{}"
    96	    if is_windows; then
    97	        wsl_integrations="{\"$WSL_DISTRO_NAME\":true}"
    98	    fi
    99	    mkdir -p "$PATH_CONFIG"
   100	    cat <<EOF >"$PATH_CONFIG_FILE"
   101	{
   102	  "version": 7,
   103	  "WSL": { "integrations": $wsl_integrations },
   104	  "containerEngine": {
   105	    "allowedImages": {

Reply:
Ok, sorry, seems like I looked at the wrong file. This `version: 7` is correct; it is written to `settings.json` which will then go through version migrations.

This is what we need for the API too because blindly updating the version number without adjusting the schema to match will eventually result in an error. Having a fixed version should remain correct as long as the app includes the corresponding migrations.

Reply:
Right, I missed that.